docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,62 @@
+# Run Antrea agent simulator
+
+This Documentation describes how to run Antrea agent simulator.
+
+## Build the images
+
+1. Build simulator image
+
+   ```bash
+   make build-scale-simulator
+   ```
+
+2. Build the kubemark image
+
+   If you alreay have a kubemark image, please ignore this step.
+
+   ```bash
+   cd $KUBERNETES_PATH
+   make WHAT=cmd/kubemark KUBE_BUILD_PLATFORMS=linux/amd64
+   cp ./_output/local/go/bin/linux_amd64/kubemark cluster/images/kubemark
+   cd cluster/images/kubemark
+   docker build -t kubemark:latest .
+   ```
+
+## Create the yaml file
+
+   This demo uses 1 simulator, this command will create a yaml file build/yamls/antrea-simulator.yml
+
+   ```bash
+   make manifest-scale
+   ```
+
+   The above yaml will create one simulator Node/Pod, to change the number of instances, you can modify `spec.replicas` of the StatefulSet `antrea-agent-simulator` in the yaml, or scale it via `kubectl scale statefulset/antrea-agent-simulator -n kube-system --replicas=<COUNT>` after deploying it.
+
+## Taint the simulator node
+
+   In case that we do not want some pods run on the simulator node on the node which can use kubectl
```

Comment:
I suggest: "To prevent Pods from being scheduled on the simulated Node(s), you can use the following taint"